src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static Scanner sc;
+    private static List<Task> list;
```

Comment:
Perhaps a better variable name than list here to indicate what the list holds?
```suggestion
    private static List<Task> tasks;
```